src/steamship/agents/react/react.py

```diff
@@ -79,6 +82,9 @@ def next_action(self, context: AgentContext) -> Action:
             chat_history=self.messages_to_prompt_history(
                 messages=context.chat_history.select_messages(self.message_selector)
             ),
+            search_history=self.search_to_search_history(
```

Comment:
Could we use the same interface as `messages_to_prompt_history`?

Reply:
Done